src/views/about/Overview.vue

```diff
@@ -1,20 +1,29 @@
 <template>
   <Section>
+    <!-- high-level description of app and api -->
     <p>
-      My Geneset provides a way for users to conveniently collect, save, and
-      share genes from 20+ different species. You can use it through this
-      website or programmatically through the
+      MyGeneset allows scientists to conveniently collect, save, and share genes
```

Comment:
I noticed that FAQ page uses `My Geneset`, but this page uses `MyGeneset`. We should make it consistent. (I don't have a strong preference on which one should be used.)